### ISSUE TITLE
Null object issues with no-private-func rule

### ISSUE THREAD

**Issue by loopyDeveloper** [willowtreeapps/wist#64]
Issue with node.children being null when using "no-private-func" rule.

https://github.com/willowtreeapps/wist/blob/d62291faa3a24a836139757a1c7c0d2664982fab/lib/rules/no-private-func.js#L11

```
Cannot read property '1' of null
TypeError: Cannot read property '1' of null
    at isFunctionInvocation (/usr/local/lib/node_modules/@willowtreeapps/wist/lib/rules/no-private-func.js:11:42)
    at Linter.expression:exit (/usr/local/lib/node_modules/@willowtreeapps/wist/lib/rules/no-private-func.js:37:22)
    at emitOne (events.js:96:13)
    at Linter.emit (events.js:191:7)
    at BrightScriptEventListener.exitExpression (/usr/local/lib/node_modules/@willowtreeapps/wist/parser/BrightScriptEventGenerator.js:261:22)
    at ExpressionContext.exitRule (/usr/local/lib/node_modules/@willowtreeapps/wist/parser/antlr/BrightScriptParser.js:5439:18)
    at ParseTreeWalker.exitRule (/usr/local/lib/node_modules/antlr4/tree/Tree.js:212:6)
    at ParseTreeWalker.walk (/usr/local/lib/node_modules/antlr4/tree/Tree.js:195:8)
    at ParseTreeWalker.walk (/usr/local/lib/node_modules/antlr4/tree/Tree.js:193:9)
    at ParseTreeWalker.walk (/usr/local/lib/node_modules/antlr4/tree/Tree.js:193:9)
```

Sample File:
```
function sampleFunction() as object
    if GetGlobalAA().user = invalid then
        user = {
            _name : "",
        }
        GetGlobalAA()["user"] = user
    end if
    return invalid
end function
```
The dangling comma in `_name : "",` 

Maybe we can add new rules for trailing commas?
https://eslint.org/docs/rules/comma-dangle

**Comment by nishtahir** [comment]
I'm working on a fix for this in #66. I've also opened a new ticket #65 to address the trailing comma rule separately.

**Comment by nishtahir** [comment]
This issue has been addressed in #66
